pwndbg/file.py

```diff
@@ -134,3 +135,18 @@ def readlink(path):
     data = data[:-len(ending)]
 
     return data
+
+def file(path):
```

Comment:
Agree with @zachriggle.

To be honest I would even go with moving all `gdb` related stuff into its own module.
So it would be easier to change gdb into other debugger (e.g. lldb - heard it has much better python api) in the future, if needed.